data/tlg4170/tlg001a/__cts__.xml

```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg4170" xml:lang="grc" urn="urn:cts:greekLit:tlg4170.tlg001a">
+  <ti:title xml:lang="lat">Vita Ambrosiana</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg4170.tlg001a.1st1K-grc1" workUrn="urn:cts:greekLit:tlg4170.tlg001a.1st1K-grc1">
+    <ti:label xml:lang="lat">Vitae Pindari et varia de Pindaro</ti:label>
+    <ti:description xml:lang="mul">Vitae Pindari et varia de Pindaro, Drachmann, Teubner, 1903</ti:description>
```

Comment:
The `<label>` element and the `<description>` element should be changed a bit here. The label should simply be the title of the work as represented in the editions, i.e., "Vita Ambrosiana". Then add to the beginning of the description `Vita Ambrosiana, Pindar Scholia`, i.e., the title of the work and the author.